src/lane_manager.cpp

```diff
@@ -59,4 +61,5 @@ void LaneManager::add_car()
 void LaneManager::turn_car(direction dir)
 {
   m_lanes[dir]->turn_car();
+  m_ai->make_villains_decide(m_lanes);
```

Comment:
This exits very quickly if no villains are in the second positions of their lane,